discuss/advanced/emailtemplate.rst

```diff
@@ -0,0 +1,57 @@
+===============
+Email Templates
+===============
+
+A nice written email can mean the difference between total disregard of the message or an excited
+reply. And not just that, it lets you save time, stay consistent, and deliver personalized messages,
+allowing customers and prospects to have the best brand experience.
```

Comment:
This doesn't make sense. 

A nice written email doesn't make you save time. The template does, but there's no talk about templates before so it's confusing. 

I would keep some of it but rework it to include that it's about templates.